src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -3355,7 +3357,8 @@ public override BoundNode VisitDefaultExpression(BoundDefaultExpression node)
 
         public override BoundNode VisitUnconvertedObjectCreationExpression(BoundUnconvertedObjectCreationExpression node)
         {
-            throw ExceptionUtilities.Unreachable();
```

Comment:
>throw ExceptionUtilities.Unreachable(); [](http://example.com/codeflow?start=12&length=39)

I think the whole point is to prevent ```BoundUnconvertedObjectCreationExpression``` from escaping initial binding, and that is the kind of the change I would expect instead.

Reply:
Thanks, fixed.